### ISSUE TITLE
Permanent bug on nexcloud app. Not possible reset app despite the restarting of smartphone 

### ISSUE THREAD

**Issue by Eneledum** [nextcloud/android#8760]
### Steps to reproduce
1. create a nextcloud login_base
2. Remain connected with login _base on Android app
3. create a SSO login (any SSO I suppose) from an other support as a laptop
4. Delete login_base
5. Bug remains on Android app

### Expected behaviour
- Nextcloud should present a login connection page 

### Actual behaviour
- Nextcloud app remains blocked on bug

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 
Brand: HUAWEI
device: HWVOG
Model: VOG-L29
Id: HUAWEIVOG-L29
Product: VOG-L29EEA

Stock or customized system:

Nextcloud app version:
ID: com.nextcloud.client
Version: 30160190
Build flavor: gplay

###error on android
Nextcloud log here

************ CAUSE OF ERROR ************

java.lang.NullPointerException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkNotNullParameter, parameter password
	at okhttp3.Credentials.basic(Unknown Source:8)
	at okhttp3.Credentials.basic$default(Credentials.kt:28)
	at okhttp3.Credentials.basic(Unknown Source:2)
	at com.owncloud.android.lib.common.OwnCloudClientFactory.createNextcloudClient(OwnCloudClientFactory.java:217)
	at com.owncloud.android.operations.RefreshFolderOperation.updatePredefinedStatus(RefreshFolderOperation.java:338)
	at com.owncloud.android.operations.RefreshFolderOperation.updateCapabilities(RefreshFolderOperation.java:311)
	at com.owncloud.android.operations.RefreshFolderOperation.updateOCVersion(RefreshFolderOperation.java:277)
	at com.owncloud.android.operations.RefreshFolderOperation.run(RefreshFolderOperation.java:230)
	at com.owncloud.android.lib.common.operations.RemoteOperation.run(RemoteOperation.java:359)
	at java.lang.Thread.run(Thread.java:929)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30160190
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWVOG
Model: VOG-L29
Id: HUAWEIVOG-L29
Product: VOG-L29EEA

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 10.1.0.150C431

**Comment by AndyScherzinger** [comment]
Duplicate of #8370
